source/blog/2017-04-27-rspec-3-6-has-been-released.md

```diff
@@ -0,0 +1,144 @@
+for better readability. In 3.1 we added the ability to define
+[http://rspec.info/blog/2014/09/rspec-3-1-has-been-released/#expectations-new-definenegatedmatcher-api](negated matchers)
+with the `define_negated_matcher` method.
+
+Before RSpec 3.6 when you called either of these methods the newly defined
+matchers were always defined at the global scope. With this feature you are able
+to invoke either `alias_matcher` or `define_negated_matcher` in the scope of an
+example group (`describe`, `context`, etc). When doing so the new matcher that
+you define will **only** be available for the duration of that example group:
+
+
+~~~ ruby
+RSpec.describe 'scoped matcher aliases' do
+  describe 'example group with a matcher alias' do
+    alias_matcher :be_a_string_starting_with, :start_with
+
+    it 'can use the matcher alias' do
+      expect('a').to be_a_string_starting_with('a')
+    end
+  end
+
+  describe 'example group without the matcher alias' do
+    it 'cannot use the matcher alias' do
+      # fails because the matcher alias is not available
+      expect('a').to be_a_string_starting_with('a')
+    end
+  end
+end
+~~~
+
+### Mocks: `without_partial_double_verification`
+
+When we released RSpec 3.0 we added [verifying doubles](http://rspec.info/blog/2014/05/notable-changes-in-rspec-3/#verifying-doubles).
+Verifying doubles allow you to ensure that stubs and mocks that you create with
+RSpec correctly emulate the interfaces on the objects in your tests.
+`without_partial_double_verification` allows you to turn off the verifying
+double behaviour for the duration of the execution of a block. For example:
+
+~~~ ruby
+class Widget
+  def call(takes_an_arg)
+  end
+end
+
+RSpec.describe Widget do
+  it 'can be stub with a mismatched arg count' do
+    without_partial_double_verification do
+      w = Widget.new
+      allow(w).to receive(:call).with(1, 2).and_return(3)
+      w.call(1, 2)
+    end
+  end
+end
+~~~
+
+Here, this test would fail if the `without_partial_double_verification` call was
+not made, because we are stubbing the `call` method on the `Widget` object with
+an argument count that is different to the implementation. We added this feature
+specifically to address a problem with partial double verification when stubbing
+locals in views. Details can be found in [this issue](https://github.com/rspec/rspec-mocks/issues/1102)
+and the rspec-rails issues linked from it.
+
+A special thanks to Jon Rowe for adding this feature.
+
+### Rails: Support for Rails 5.1:
+
+RSpec 3.6.0 comes with full support for Rails 5.1. There are no major changes to
+the rails 5.1 API and so this upgrade should be fully smooth. Simply bumping to
+the latest version of RSpec will bring the support to your app with no other
+changes required on your part.
+
+Rails [system tests](http://weblog.rubyonrails.org/2017/4/27/Rails-5-1-final/) are not yet supported,
+but we plan to add support for them in the near future.
```

Comment:
Only in that they're "blessed" and use a specific non-forked process server (run with puma) that allows for transactional testing.